### PR TITLE
Implement NuGetRepack tool

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by jaredpar** [comment]
Is this the tool that allows for a NuPkg to be repackaged as the different flavors: release, perbuild release, etc ...?

File: src/NuGetRepack/Program.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        private static int Main(string[] args)
+        {
+            var operation = Operation.None;
+            var packages = new List<string>();
+            string outDirectory = null;
+
+            try
+            {
+                int i = 0;
+                while (i < args.Length)
```
Comment (jaredpar) [inline src/NuGetRepack/Program.cs]: Consider using Mono.options for the command line arguments. I've been using it for a lot of tools recently. Really simplifies the parsing and printing of help messages.

File: src/NuGetRepack/Program.cs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                }
+            }
+            catch (AggregateException e)
+            {
+                foreach (var inner in e.InnerExceptions)
+                {
+                    Console.Error.WriteLine(inner.Message);
+                }
+
+                return 3;
```
Comment (jaredpar) [inline src/NuGetRepack/Program.cs]: Consider giving names to magic numbers.

File: src/NuGetRepack/Program.cs (hunk 1 above)
Comment (tmat) [reply src/NuGetRepack/Program.cs]: I don't think there is any benefit of adding extra dependency on Mono.Options. This is very trivial code. BTW, Mono.Options is not signed which is causing problems.

**Comment by tmat** [comment]
Yes, that's exactly what it is.

File: src/NuGetRepack/Program.cs (hunk 2 above)
Comment (tmat) [reply src/NuGetRepack/Program.cs]: ok.

**Pull request merged**